packages/app/src/App.tsx

```diff
@@ -12,10 +12,22 @@ import { AsyncStorageExample } from "./AsyncStorageExample";
 import { subplatform } from "./config";
 import LogoSrc from "./logo.png";
 
+// Conditionally import the TV specific react-native types
+// Note: the eval is necessary to avoid static analysis errors
+(async () => {
+  if (Platform.isTV) {
+    eval("import('../../tv/node_modules/react-native/tvos-types.d');");
+  }
+})();
+
 export function App(): JSX.Element {
-  const platformValue = subplatform
-    ? `${Platform.OS} (${subplatform})`
-    : Platform.OS;
+  let platformValue;
+  if (Platform.isTV) {
+    platformValue = Platform.OS === 'ios' ? 'tvos' : 'android tv';
```

Comment:
Can we keep the `platform.OS` reference as it is and put `tvos` and `android tv` in `subplatform` instead? 
My thinking here is that the `Platform.OS` should be shown and used "as it is" (because it's a RN API that shows the OS used) and use subplatform as the variant. 
E.g.:
```js
// config.ts

import { Platform } from "react-native";

// eslint-disable-next-line
declare var __SUBPLATFORM__: "electron" | "browser-ext" | "android-tv" | "tvos" | undefined; 

export const isDev = __DEV__;

// Tells on what variant of the platform we're running
export let subplatform = undefined;

// Injected in electron and browser-extension builds.
if (typeof __SUBPLATFORM__ === "string") {
  subplatform = __SUBPLATFORM__
}
// For tvOS and Android TV, we can check the Platform.isTV field 
else if (Platform.isTV && Platform.OS === "ios") {
  subplatform = "tvos";
} else if (Platform.isTV && Platform.OS === "android") {
  subplatform = "android-tv";
}
```

Let me know if it makes sense and/or if you agree with this 👍

Reply:
Yeah makes sense -- I wish I could inject it with Babel but I couldn't find a way to differentiate between iOS and Android.